app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,5 @@
+module SessionsHelper
+	def current_user
+		User.find_by(id: session[:user_id])
+	end
+end
```

Comment:
@saki4567 Please add here a new method for log_in:

```
def log_in(user)
  session[:user_id] = user.id
end
```
After that, go to **application_controller.rb**, and add these statement inside class:
`include SessionsHelper`


Every time you want the user to log in, you can use this method.